src/api/account.ts

```diff
@@ -109,7 +110,18 @@ export class Account {
     moduleName: string;
     options?: LedgerVersion;
   }): Promise<MoveModuleBytecode> {
-    const module = await getModule({ aptosConfig: this.config, ...args });
+    // We don't memoize the account module by ledger version, as it's not a common use case, this would be handled
+    // by the developer directly
+    if (args.options?.ledgerVersion !== undefined) {
```

Comment:
I think we should memoize the internal functions. other sdk functions use the internal queries

Reply:
Done